test-case/verify-sof-firmware-load.sh

```diff
@@ -33,5 +33,6 @@ if $cmd | grep -q " sof-audio.*version"; then
     $cmd | grep "Firmware debug build" -A3 | head -n 12
     exit 0
 else
+    journalctl --dmesg |tail -50
```

Comment:
`journalctl` has a (faster) built-in `--lines` option.

Print the value of `$cmd` too, bugs in `sof-kernel-dump.sh` are very likely considering how ugly the code is so we want to know which one ran.
```shell
journalctl --dmesg --lines 50
journalctl --dmesg | grep -C 1 -i version
printf '$cmd was %s\n' "$cmd"
```

Reply:
thanks for the suggestion, will update.